samples/OpenThread/UdpThreadServer/Program.cs

```diff
@@ -0,0 +1,148 @@
+using System;
+using System.Net;
+using System.Threading;
+using nanoFramework.Networking.Thread;
+
+namespace Samples
+{
+    public class Program
+    {
+        const int UDP_PORT = 1234;
```

Comment:
same remarks as for the other sample, we should be explicit with the visibility and underscore for private global variables.